.github/test-spec.yml

```diff
@@ -1,4 +1,7 @@
 # This is the Jenkins ci variant of the .github/labler.yaml
+"CI-iot-zephyr-lwm2m-test":
+  - "**/*"
```

Comment:
I think this is too wide.

I would suggest for example:
```
drivers/console/**/*
drivers/flash/**/*
subsys/dfu/boot/**/*
subsys/net/ip/**/*
subsys/net/lib/http/**/*
subsys/net/lib/lwm2m//**/*
subsys/net/**/*
```

Reply:
Updated.